packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json

```diff
@@ -2,12 +2,16 @@
   "user_account_identity_documents": "Mes documents d'identité",
   "user_account_identity_documents_sub_title": "Déposez vos documents",
   "user_account_identity_documents_intro": "Vous avez créé votre compte et vous souhaitez faire une commande depuis notre site Indien. La règlementation en vigueur nécessite une vérification de votre identité avant toute commande.",
-  "user_account_identity_documents_list_for_particular": "Vous êtes un particulier, les documents obligatoires à fournir sont Carte d'identité ou passeport (ou permis de conduire, carte électorale, preuve de possession d'un nombre Aadhaar), Justificatif de domicile.",
-  "user_account_identity_documents_list_for_enterprise": "Vous êtes une entreprise, les documents obligatoires à fournir sont Carte d'identité ou passeport (ou permis de conduire, carte électorale, preuve de possession d'un nombre Aadhaar), Justificatif de domicile, Certificat GST.",
-  "user_account_identity_documents_list_for_association": "Vous êtes une association, les documents obligatoires à fournir sont Carte d'identité ou passeport (ou permis de conduire, carte électorale, preuve de possession d'un nombre Aadhaar), Justificatif de domicile, Certificat GST (optionnel).",
-  "user_account_identity_documents_list_for_institution": "Vous êtes une association, les documents obligatoires à fournir sont Carte d'identité ou passeport (ou permis de conduire, carte électorale, preuve de possession d'un nombre Aadhaar), Justificatif de domicile, Certificat GST (optionnel).",
-  "user_account_identity_documents_list_for_others": "Voici les documents obligatoires à fournir sont Carte d'identité ou passeport (ou permis de conduire, carte électorale, preuve de possession d'un nombre Aadhaar), Justificatif de domicile, Certificat GST (optionnel).",
-  "user_account_identity_documents_descriptiom": "Les documents seront conservés durant 5 ans après la fin du contrat.",
+  "user_account_identity_documents_list_for_particular": "Vous êtes un particulier, les documents obligatoires à fournir sont",
+  "user_account_identity_documents_list_for_enterprise": "Vous êtes une entreprise, les documents obligatoires à fournir sont",
+  "user_account_identity_documents_list_for_association": "Vous êtes une association, les documents obligatoires à fournir sont",
+  "user_account_identity_documents_list_for_institution": "Vous êtes une association, les documents obligatoires à fournir sont",
+  "user_account_identity_documents_list_for_others": "Voici les documents obligatoires à fournir sont:",
```

Comment:
It should be `"user_account_identity_documents_list_for_others": "Les documents obligatoires à fournir sont:",` **or** `"user_account_identity_documents_list_for_others": "Voici les documents obligatoires à fournir:",`

Reply:
done Thanks !